kernel/src/main/scala/cats/kernel/Order.scala

```diff
@@ -139,7 +139,16 @@ abstract class OrderFunctions[O[T] <: Order[T]] extends PartialOrderFunctions[O]
     ev.max(x, y)
 }
 
-object Order extends OrderFunctions[Order] {
+trait OrderToOrderingConversion {
+  /**
+   * Implicitly derive a `scala.math.Ordering[A]` from a `Order[A]`
+   * instance.
+   */
+  implicit final def catsKernelOrderingForOrder[A](implicit ev: Order[A]): Ordering[A] =
+    ev.toOrdering
+}
+
+object Order extends OrderFunctions[Order] with OrderToOrderingConversion {
```

Comment:
I haven't researched this, but I am wondering if we place the method directly in the object instead of a trait, will that help binary comp going forward (on scala 2.11.x)?